pyoxidizer/src/py_packaging/distribution.rs

```diff
@@ -333,6 +350,102 @@ pub enum ExtensionModuleFilter {
     NoGPL,
 }
 
+pub struct PythonPaths {
+    pub prefix: PathBuf,
+    pub bin_dir: PathBuf,
+    pub python_exe: PathBuf,
+    pub stdlib: PathBuf,
+    pub site_packages: PathBuf,
+    pub pyoxidizer_state_dir: PathBuf,
+}
+
+/// Resolve the location of Python modules given a base install path.
+pub fn resolve_python_paths(base: &Path, python_version: &str) -> PythonPaths {
+    let prefix = base.to_path_buf().canonicalize().unwrap();
```

Comment:
`canonicalize()` on Windows does horribly things and we should generally avoid calling it. It is best to instead pass in an absolute path and rely on normal path manipulation functions to _just work_. I will likely drop the use of `.canonicalize()` as part of landing this.